src/main.c

```diff
@@ -130,15 +137,26 @@ static void load_port_config(struct uci_context *uci, struct uci_section *s)
 
 static void load_global_config(struct uci_context *uci, struct uci_section *s)
 {
-	const char *budget, *guardband;
+	const char *budget, *guardband, *i2c_bus, *i2c_addr;
```

Comment:
As with my previous comment, if we can pull these from the device tree, there'd be no need to read this data in from  configuration file.

Reply:
Indeed.